Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferParserAVFObjC.mm

```diff
@@ -250,43 +251,50 @@ AtomString codec() const override
 
 void SourceBufferParserAVFObjC::flushPendingMediaData()
 {
+    INFO_LOG_IF_POSSIBLE(LOGIDENTIFIER);
     [m_parser providePendingMediaData];
 }
 
 void SourceBufferParserAVFObjC::setShouldProvideMediaDataForTrackID(bool should, uint64_t trackID)
 {
+    INFO_LOG_IF_POSSIBLE(LOGIDENTIFIER, "should = ", should, ", trackID = ", trackID);
     BEGIN_BLOCK_OBJC_EXCEPTIONS
     [m_parser setShouldProvideMediaData:should forTrackID:trackID];
     END_BLOCK_OBJC_EXCEPTIONS
 }
 
 bool SourceBufferParserAVFObjC::shouldProvideMediadataForTrackID(uint64_t trackID)
 {
+    INFO_LOG_IF_POSSIBLE(LOGIDENTIFIER, "trackID = ", trackID);
     return [m_parser shouldProvideMediaDataForTrackID:trackID];
 }
 
 void SourceBufferParserAVFObjC::resetParserState()
 {
+    INFO_LOG_IF_POSSIBLE(LOGIDENTIFIER);
     m_parserStateWasReset = true;
 }
 
 void SourceBufferParserAVFObjC::invalidate()
 {
+    INFO_LOG_IF_POSSIBLE(LOGIDENTIFIER);
     [m_delegate invalidate];
     m_delegate = nullptr;
     m_parser = nullptr;
 }
 
 #if !RELEASE_LOG_DISABLED
-void SourceBufferParserAVFObjC::setLogger(const Logger& logger, const void* logIdentifier)
+void SourceBufferParserAVFObjC::setLogger(const Logger& newLogger, const void* newLogIdentifier)
```

Comment:
don't think we need to rename those arguments, that's not common naming style for setLogger

Reply:
it causes a naming conflict with the logging macro, since the macro calls `logIdentifier()`, the local variable is used instead which doesn't build.